lib/app/views/partials/section.html

```diff
@@ -50,7 +50,7 @@ <h1 class="sg">
         <i class="fa fa-close"></i>
       </a>
     </div>
-    <pre class="sg" ng-if="section.showCSS"><code hljs source="section.css" class="html"></code></pre>
+    <pre class="sg" ng-if="section.showCSS"><code hljs hljs-language="css" hljs-source="section.css" class="html"></code></pre>
```

Comment:
`hljs` seemed to have trouble identifying the source language so I added language attributes. In the current version, it is selecting languages like `axapta` and `applescript`. It was also selecting the language inconsistently between markup sections on the page. This should fix all of the above.